tests/NLog.UnitTests/ConfigFileLocatorTests.cs

```diff
@@ -50,7 +50,7 @@ public class ConfigFileLocatorTests
         private string appConfigContents = @"
 <configuration>
 <configSections>
-    <section name='nlog' type='NLog.Config.ConfigSectionHandler, NLog' requirePermission='false' />
+    <section name='nlog' type='NLog.Config.XmlLoggingConfigurationSection, NLog' requirePermission='false' />
```

Comment:
This seems not be backwards compatible. 

I think we should rename the class back to `ConfigSectionHandler`